README.md

```diff
@@ -170,11 +192,17 @@ Plot count of different reactions to posts
 
 ## Observations
 
-1. There is a spike in friends made in March (Election season) and July (new juniors, much higher spike).
 
-2. I tend to message less during exams (Feb, Apr, Sep, Nov).
 
-3. Highest number of messages sent at 9 and 11 pm, confirming with calls from home come at 10pm. Almost no messages shared between 3am-7am.
+1. In the first month I haven't received any friend request.
```

Comment:
Please do not add your observations to the top, add them at the end.

Reply:
Done